### DISCUSSION TITLE
Float height 2.2.5 vs 2.4.1/2.4.2

### DISCUSSION THREAD

**Discussion by Neoony** [zolantris/ValheimMods#157] [General]
Not sure if this is an issue, or just something that is now different

It seems after update to 2.4.1 or 2.4.2 the ships float lower

This ship only had water inside in some storms
https://imgur.com/VyGxjPS

But now the first level is full of water

Its true I only use hull base in the middle of it on bottom and the rest is just flat slags
(the water vehicle piece just destroyed itself, just like it did on all our ships, so we just replaced with hulls and that works)

https://i.imgur.com/JbIH47K.jpeg

Would adjusting some of the new settings revert this back?

Not sure what the values were in the old version, since it was not in config

And I dont want a fixed offset

**Comment by Neoony** [comment]
additionally

in this issue

https://github.com/zolantris/ValheimMods/issues/152

it was mentioned
"Noting that all the damping values on water surface should be 0.5 to be safe but realistic..but I set it to 0.05 which was way too low during publish."

However the new version has 

```
## Controls how much the water pushes the boat upwards directly. This value may affect angular damping too. Recommended to keep the original value. But tweaking can remove or add additional jitter. Higher values likely will add more jitter.
# Setting type: Single
# Default value: 1
# Acceptable value range: From 0.05 to 1
waterDamping_2.4.2 = 1

## Controls how much the water pushes the boat sideways based on wind direction and velocity.
# Setting type: Single
# Default value: 2
# Acceptable value range: From 0.5 to 5
waterSidewaysDamping_2.4.2 = 2

## Controls how much the water pushes the boat from a vertical angle based on water and velocity. Lower values will cause more rocking and allow better turn rates. Higher values will make the vehicle more stable, but less turning angle and possibly less realistic. If you get motion-sickness this can allow tweaking sway without disabling it all and also prevent rapid turning.
# Setting type: Single
# Default value: 1
# Acceptable value range: From 0.05 to 1
waterAngularDamping_2.4.2 = 1
```

is that right?

**Reply by zolantris** [reply]
Yeah I went overboard here. It's probably better to drop things to 0.3 or 0.5 but throttle high speed turns. Something to considered in upcoming releases.

**Comment by Neoony** [comment]
Similarly if I just start a new world and place water vehicle, it just sinks under the water and floats a bit under the water level
![image](https://github.com/user-attachments/assets/da1baf7f-49e4-4aa6-8ed0-d22e08ffaa2e)

I dont want to have to build a big bottom to make it float higher (since that will get stuck on rocks)

tried to mess with waterDamping_2.4.2 in single player

but it does not seem to make any difference for this. 1 or 0.05 seems the same

**Comment by Neoony** [comment]
Placing hulls on top of it, or on bottom, actually does nothing at all

![image](https://github.com/user-attachments/assets/617e1c65-a9ea-4fd3-a99d-32d38c46647a)

while
HullFloatationColliderLocation = Average

**Reply by zolantris** [reply]
Something is wrong with average. I have to fix that. Low priority so things were set to custom offset (from ship sinking feedback) as people were  surprised their ship sinks as they place items higher.

**Comment by Neoony** [comment]
I guess one way is to enable 
WaterBallastEnabled = true
and set
AboveSurfaceBallastMaxShipSizeAboveWater = 1
(hmm weird, 1 seemed to be fine, but then something happened and 1 was waaay too much and I had to put it to 0.07, huh)

and then with space, I can get on water level again

But, seems like something is wrong with average floatation?

**Comment by jakedamico** [comment]
Is there a better way to raise the boat higher? I find my boat constantly dragging along the bottom because of how low it goes, with the water level basically rising to the top of the hull by default. Is there any way to adjust this more? I tried this method and it seemed to help but not substantially.
